packages/common/src/apiutil/LionwebResponse.ts

```diff
@@ -57,6 +57,10 @@ export interface IdsResponse extends LionwebResponse {
     ids: string[]
 }
 
+export interface ListRepositoriesResponse extends LionwebResponse {
```

Comment:
If this is called `ListrRepositoriesResponse` this seems an indication that `PartitionsResponse` should become `ListPartitionsResponse`

Reply:
Indeed.